autoload/coiledsnake.vim

```diff
@@ -45,29 +45,29 @@ function! coiledsnake#RefreshFolds() abort "{{{1
         " Open a fold at the appropriate level on this line.
         let b:marks[l:fold.lnum] = '>' . l:fold.level
 
-        " If no inside line was found, the fold reaches the end of the file and 
+        " If no inside line was found, the fold reaches the end of the file and
```

Comment:
Can you add back the trailing spaces in comment lines?  I know most people get rid of all trailing space, but I use `:set formatoptions+=w`, so these spaces help makes sure that comments lines are wrapped properly.  Feel free to remove trailing whitespace from any non-comment lines, though.

Reply:
Oh! Didn't know that! Sorry.

Adding it back..